_posts/2024-05-02-vega-tsvb-mds-visualizations.markdown

```diff
@@ -0,0 +1,155 @@
+---
+layout: post
+title:  "Creating Visualizations Leveraging Multiple Data Sources"
+authors:
+ - wronghuy
+date: 2024-05-02
+categories:
+ - technical-post
+meta_keywords: multiple data source, vega, tsvb
+meta_description: Learn how to create tsvb and vega visualizations with the multiple data sources feature.
+excerpt: In 2.13 and 2.14 release, multiple data sources (MDS) now supports Vega and TSVB. In this blog, we will explore how to leverage different OpenSearch clusters to view your data.
+---
+
+## Introduction
+In 2.4, OpenSearch Dashboards introduced the [multiple data sources feature](https://opensearch.org/blog/multiple-data-source/) (MDS), which enables users to visualize their data from different OpenSearch clusters. While MDS provided compatibility with most of Dashboards visualizations, there were a few visualization types that were not compatible. With 2.13 and 2.14 release, however, Dashboards introduces multiple data sources compatibility with two of those visualizations: Vega and TSVB.
```

Comment:
```suggestion
[Multiple data sources](https://opensearch.org/blog/multiple-data-source/) gives users the capability to visualize data from various OpenSearch clusters. To date, only certain visualization types have been available. With the release of OpenSearch Dashboards 2.14, multiple data sources is now compatible with  [Vega](https://opensearch.org/docs/latest/dashboards/visualize/vega/) and [Time-Series Visual Builder (TSVB)](https://opensearch.org/docs/latest/dashboards/visualize/viz-index/#tsvb)) visualizations types.
```

Reply:
Who is the developer that can write end user documentation for TSVB? I can support the developer in getting it published to the OpenSearch doc site.